src/Sarif/Core/Location.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public partial class Location
+    {
+        public LogicalLocation LogicalLocation
```

Comment:
>public LogicalLocation LogicalLocation [](start = 8, length = 38)

i need to add deprecated message

Reply:
[https://github.com/microsoft/sarif-sdk/issues/1495](https://github.com/microsoft/sarif-sdk/issues/1495)

---
In reply to: [285746290](https://github.com/microsoft/sarif-sdk/pull/1494#discussion_r285746290) [](ancestors = 285746290)